sigmf-spec.md

```diff
@@ -630,6 +631,7 @@ Segment Objects:
 | `comment`         | false    | string | A human-readable comment.                                                           |
 | `freq_lower_edge` | false    | double | The frequency (Hz) of the lower edge of the feature described by this annotation.   |
 | `freq_upper_edge` | false    | double | The frequency (Hz) of the upper edge of the feature described by this annotation.   |
+| `uuid`            | false    | string | A RFC-4122 compliant UUID string of the form `xxxxxxxx-xxxx-Mxxx-Nxxx-xxxxxxxxxxxx`.|
```

Comment:
Excellent. I really like having an explicit definition for this. How about adding the line
```python
 UUID_KEY = "core:uuid"
```
to sigmffile.py and appending `UUID_KEY` to `VALID_ANNOTATION_KEYS`?

Reply:
good call, thanks.